lib/galaxy/model/__init__.py

```diff
@@ -3579,8 +3601,9 @@ def __init__( self ):
         self.position = None
         self.input_connections = []
         self.config = None
+        self.label = None
         self.uuid = uuid4()
-        self.worklfow_outputs = []
+        self.workflow_outputs = []
```

Comment:
Thanks - it was a test I'm sure - with my dyslexia I really need to stop and stare for a long time even in the red and green of the diff to see the difference.